index.bs

```diff
@@ -953,33 +954,42 @@ spec: webidl
     using <code>{{Promise}}.all</code>.
   </p>
   <pre class='highlight'>
-    &lt;script&gt;
-      Promise.all([navigator.permissions.query({name:'geolocation'}),
-                   navigator.permissions.query({name:'notifications'})])
-      .then(function(result) {
-        console.log('Geolocation permission state is ' + result[0].state);
-        console.log('Notifications permission state is ' + result[1].state);
-      });
-    &lt;/script&gt;
+Promise.all([
+    navigator.permissions.query({ name: "geolocation" }),
+    navigator.permissions.query({ name: "notifications" })
+  ])
+  .then(([{ state: geoState }, { state: notifState }]) => {
+    console.log("Geolocation permission state is:", geoState);
+    console.log("Notifications permission state is:", notifState);
+  });
   </pre>
   </div>
   <div class="example" id="example-deviceId">
   <p>
-    This example is checking whether the page has the {{"camera"}}
-    permission for the first device in the list. Error checking omitted.
+    This example is checking the permission of the the first camera.
```

Comment:
s/is checking/checks/? 

Can you say what "the permission of" means. Is it permission to record video from, permission to see the metadata of, or what?

You have a "the the".